.gitignore

```diff
@@ -374,11 +374,16 @@ lightgbm.model
 *.pkl
 
 # macOS
-.DS_Store
+**/.DS_Store
 
 # VSCode
 .vscode
 
+# Files from local Python install
+python-package/LICENSE
+python-package/build_cpp/
+python-package/compile/
+
 # duplicate version file
 python-package/lightgbm/VERSION.txt
```

Comment:
I suppose `VERSION.txt` here is from local Python install too and can be moved under the same section.

Reply:
sure that's fair. I can do that

Reply:
Thanks!

Reply:
fixed